scripts/build-examples.js

```diff
@@ -61,15 +62,15 @@ function buildExample(outDir, relativeFilePath, bundleFileName) {
     }
 
     fs.outputFileSync(
-        path.join(outDir, relativeFilePath.replace('examples/', '')),
+        path.join(outDir, path.relative(examplesPath, relativeFilePath)),
```

Comment:
this one might be slightly risky.  there's no requirement that the relative path passed to this function is from our examples dir.  Though, in practice (both for dev server and building our examples for deployment/testing) this does seem to be the case.

Reply:
I understand, will change that.
The reason for this modification was with `windows` development support, `glob` sends a `relativePath` with `/` even when running on a `windows` machine, but it's not the case for `path` module.
We can keep the `replace()` method but with support replacing backslash characters.

Reply:
👍 ... assumed it was for windows dev and thanks for adding support.

And, thanks for switching to the regexp approach.  probably safest fix.